pkgs/applications/misc/stork/default.nix

```diff
@@ -1,20 +1,26 @@
 { lib
 , rustPlatform
 , fetchFromGitHub
+, openssl
+, pkg-config
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "stork";
-  version = "1.1.0";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "jameslittle230";
     repo = "stork";
     rev = "v${version}";
-    sha256 = "sha256-pBJ9n1pQafXagQt9bnj4N1jriczr47QLtKiv+UjWgTg=";
+    sha256 = "sha256-gPrXeS7XT38Dil/EBwmeKIJrmPlEK+hmiyHi4p28tl0=";
   };
 
-  cargoSha256 = "sha256-u8L4ZeST4ExYB2y8E+I49HCy41dOfhR1fgPpcVMVDuk=";
+  cargoSha256 = "sha256-9YKCtryb9mTPz9iWE7Iuk2SKgV0knWRbaouF+1DCjv8=";
+
+  nativeBuildInputs = [ openssl.dev pkg-config ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pkg-config ];
  
  buildInputs = [ openssl ];
```

Reply:
That mistake of mine explains why I had to manually set `PKG_CONFIG_PATH`, thanks. I've force-pushed with `openssl` correctly in `buildInputs` after running the same tests.